ot/bregman/_empirical.py

```diff
@@ -198,6 +239,8 @@ def empirical_sinkhorn(X_s, X_t, reg, a=None, b=None, metric='sqeuclidean',
         if log:
             dict_log["u"] = f
```

Comment:
yes we have a consistencey poblem, we should open and issue and do a quick check for all entropic slvers.. it might break API for people ut we need to do that weel (maybe after release 1.0 where all is uniform in solve....)